app.js

```diff
@@ -408,11 +410,80 @@ client.on("message", async message => {
                 message.channel.send("Discord's best practices for bots state that failed commands should fail silently");
             }
         }
-
     };
     // message.delete();
     if (message.guild.id.toString().includes(config.logserver)) {
-        client.channels.get(config.logchannel).send( message.author.username + '(' + message.author.id + ') ran command `' + message.cleanContent + '` in ' + message.channel, {"split":true});
+        client.channels.get(config.logchannel).send({embed:{
+            color: colors.green,
+            author: {
+                name : message.author.username,
+                icon_url: message.author.displayAvatarURL
+            },
+            title: "Command ran in #".concat( message.channel.name ),
+            fields: [{
+                name: "Command",
+                value: "`".concat( message.cleanContent, "`")
```

Comment:
concat is actually meant for arrays so this converts all the strings into char arrays and merges them then returns a new array. its easier and often cleaner to just use `+` between all the things you want to concatenate.

Reply:
this goes for all the `.concat` on strings, just `+` is good.